pkg/chart/chart.go

```diff
@@ -174,15 +228,24 @@ func NewTesting(config config.Configuration) Testing {
 	return testing
 }
 
-func (t *Testing) processCharts(action func(chart string, valuesFiles []string) TestResult) ([]TestResult, error) {
+func (t *Testing) processCharts(action func(chart *Chart) TestResult) ([]TestResult, error) {
 	var results []TestResult
-	charts, err := t.FindChartsToBeProcessed()
+	chartDirs, err := t.FindChartDirsToBeProcessed()
 	if err != nil {
 		return nil, errors.Wrap(err, "Error identifying charts to process")
-	} else if len(charts) == 0 {
+	} else if len(chartDirs) == 0 {
 		return results, nil
 	}
 
+	charts := []*Chart{}
+	for _, dir := range chartDirs {
+		chart, err := NewChart(dir)
+		if err != nil {
+			return nil, err
+		}
+		charts = append(charts, chart)
+	}
```

Comment:
This costs a bit more in terms of memory, but we are now able to catch configurations resulting in bad chart paths up front instead of waiting in sequence for helm install or version checks to fail.